tests/utils.py

```diff
@@ -247,11 +250,31 @@ def extract_info(response: Response) -> Dict[str, Any]:
         "status_code": response.status_code,
         "body": response.json(),
         "antiCsrf": response.headers.get("anti-csrf"),
-        "accessTokenFromHeader": response.headers.get("st-access-token"),
-        "refreshTokenFromHeader": response.headers.get("st-refresh-token"),
+        "accessTokenFromHeader": access_token_from_header,
+        "refreshTokenFromHeader": refresh_token_from_header,
+        "accessTokenFromAny": access_token_from_header or access_token,
```

Comment:
Does this handle empty strings? We are only checking for `is not None` right now, but in the future, this could cause issues:
- if `access_token_from_header` is an empty string (because it cleared the token), it'll return `access_token` (None in case we cleared only using headers)
- checking for `is None` thinking it'd check that neither added an update passes even if we cleared using headers
- checking for an empty string, thinking it'd check that one of them cleared the token but it fails even though it was cleared using headers.